stitches/tests/test_internal_data.py

```diff
@@ -3,65 +3,57 @@
 import os
 import unittest
 
+from stitches.fx_util import nrow
 from stitches.fx_pangeo import fetch_pangeo_table
 
-class MyInternalData(unittest.TestCase):
+class TestInternalData(unittest.TestCase):
 
+    # This test checks the internal package data and throws an error if the
+    # internal package data needs to be updated/rerun.
+    PANGEO_TABLE = pd.read_csv(pkg_resources.resource_filename('stitches', 'data/pangeo_comparison_table.csv'))
+    MATCHING_ARCHIVE = pd.read_csv(pkg_resources.resource_filename('stitches', 'data/matching_archive.csv'))
 
-    def test_pkg_data(self):
+    # Get the most up todate pangeo catalog
+    CURRENT_DATA = fetch_pangeo_table()
 
-        # Make sure the matching data is consistent with internal tas data.
-        #
-        # Read in the archive that is used in the matching process, we are going to
-        # compare the model / experiment / ensembles / variables that exist here with
-        # must also exist in the raw tas data files.
-        path = pkg_resources.resource_filename('stitches', 'data/matching_archive.csv')
-        df = pd.read_csv(path)[["experiment", "model", "ensemble"]]
-        matching_archive = df.drop_duplicates(keep='last')
+    def test_pangeo_catalog1(self):
+        internal_fresh = self.PANGEO_TABLE.merge(self.CURRENT_DATA, how='left', indicator=True)
+        new_data = internal_fresh[internal_fresh["_merge"] == 'left_only']
```

Comment:
@crvernon is it possible to have two assert `assertEqual` tests in one unit test function? Or does only one get evaluated?